### ISSUE TITLE
Problem with displaying the "event" and "access_time" icons

### ISSUE THREAD

**Issue by d4n5h** [darrenfang/vuetify-datetime-picker#95]
![exampleee](https://user-images.githubusercontent.com/1638614/115654872-3a939000-a35c-11eb-8dbd-d7e7fa47e7be.png)

**Comment by SUTFutureCoder** [comment]
I also have the same problem. Please try to **use custom icons** and **add 'mdi-'** before your icon name.

Here is my code.

```
<v-datetime-picker label="Select Datetime" v-model="time">
        <template slot="dateIcon">
          <v-icon>mdi-calendar</v-icon>
        </template>
        <template slot="timeIcon">
          <v-icon>mdi-clock-outline</v-icon>
        </template>
</v-datetime-picker>
```

reference: 
https://github.com/darrenfang/vuetify-datetime-picker/blob/master/src/app.vue
custom icons

**Comment by d4n5h** [comment]
@SUTFutureCoder Yeah that's what I ended up doing. But then I saw that this library doesn't support SSR and validation so I created my own Datetime picker component that supports both. I can publish it if anyone wants it.

**Comment by d4n5h** [comment]
@SUTFutureCoder https://gist.github.com/d4n5h/a5c55cd1dfb71dda575946dc8ccf4acb

**Comment by wackyapps** [comment]
@d4n5h thanks for sharing.
